### ISSUE TITLE
Is an alternative account an issue, and how to deal with it if so.

### ISSUE THREAD

**Issue by fcaps** [FAForever/server#1001]
TL;DR
FAF has a ruling against using multiple accounts, but is this actually helping the community?

Using an alt account can have multiple reasons, like using it for streaming, or maybe you just don't wanna play with your friends today.
I could list multiple reasons here, but let's focus on what FAF is trying to counter with the rule "only one account per person":
Smurfing! Smurfing is an issue for multiple games for a number of reasons, but what is hurting our community really?

1. Alt-Accounts in general?
2. Smurfing?
3. Trolling?
4. Win-Trading?
5. Lobby Sim?
6. Lag / Network issues?
7. Salty Players?
8. Players how will/can not communicate in English?
9. Hosts that are afk when the lobby is full?
10. High entry bar for new players who will leave after a couple of games?
11. A lobby server that is loosing the state of my selected faction in tmm?
12. A game that will segfault if i type something in chat during "tmm game found"
13. Community members who will always find something against every new "idea" from other players.
14. TMM not beeing marketed, so players still don't know that this exists.
15. No in-game mute option?
16. A rating system that is mostly useless on a game with multiple map-types and players specialized on them?
17. ...

Maybe i am getting a lil salty here, but trying to counter smurfing with a "one account" rule feels odd when the RTS genre has so many other issues we even didn't start working on.

To make any assumption on how damaging Alt-Accounts are, we need to collect data:
1. Flag games with alt-accounts, so we have a list of them an can investigate how they are used.
2. Are the other players annoyed bc. of that alt-account?
3. Do players stop playing bc. they lost to an alt account?
4. Why are ppl annoyed playing against an alt-account?
5. ...

Since the playerbase is not that huge, i would even say: Fuck it, make a feature in the website to create an alt-account for 5€, and let's see how much it hurts or even benefits the community.

You still reading all this? Okay, then let's throw some ideas on the table.

1. Collect data out of the replay-commands (probably needs to be closed source) and boost them up fast, so they can only play a few games per account in a lower tier.
2. Make tmm/ranked games only possible after some placement matches, tutorial, maybe even after beating some bot games, so creating a smurf account will introduce more pain than joy for a pro player, but will not hurt new players.
3. Flag potential smurfs with a badge "super talent of the year", so game hosts can deal with em/ balance correctly.
4. Show players a notification after they lost to a potential smurf, that the game will probably be unrated and they don't lost points.
5. As a last resort: enable two factor auth for tmm, if this is really really really super hurtful for the community.

Cheers

**Comment by IndexLibrorumProhibitorum** [comment]
I'm going to summarize your text, and then address it point by point. Please let me know if I mischaracterize your opinions or have not understood what you are trying to communicate.

You seem to argue that the rule against multiple accounts should not exist, because 
1) People may have valid reasons to have a second account,
2) There are many other issues with FAF that are more important than smurfing,
3) We need to gather data first, to see if smurfing is a real issue,
4) There are alternate ways to address the problem

Re: 1)
Can agree that smurfing hurts the community? If yes, then I am sure you agree that banning second accounts is an easy and effective method. I agree that this means that people who want a second account not for smurfing but to play on their real rating might not get one. However, I have not been shown a good reason to believe that people need more than one account, except for some particular circumstances. 

You name a few examples of such circumstances. The only one that I believe is legitimate is the example of a well known streamer requiring a second account, so that they won't be harassed during their games. You are likely unaware, but we have made exceptions to the rule in the past for exactly those cases. However, many of your other examples do not feel relevant to me. I do not think it is too much to expect, for example, that people can communicate with their friends and tell them they want to play without them sometimes. Requiring a second account for something like that, especially if it means that we lose an effective method against smurfing, is rather silly.

In short: most people do not have a valid reason to need a second account.

Re: 2)
I am not going to address all the other issues you laid out, because frankly they are not relevant to this discussion. Arguing that those issues exists and that therefore it is not important to take action against smurfing is an example of the fallacy of relative privation (or 'appeal to worse problems'). It makes no sense: the issues are separate, and require separate actions to mitigate them. Action taken against one issue does not detract from any action taken against the other issues.

I also dismiss out of hand your suggestion that the other issues haven't been addressed. I have to admit that I find you dismissing the hard work that Sheikah et al. have put into stabilizing the servers and addressing the network issues in the past months somewhat offensive. Others, such as Win-trading, trolling, smurfing, alt-accounts in general and toxicity are actively being addressed daily by our team of moderators. 

Re: 3)
To answer the first point you raised:
We regularly get reports for smurfing, where suspicious accounts play significantly out of their rating. In some cases, nothing seems to be the case. It happens that people are simply wrong about what kind of skill may be expected for a certain rating, and the reported account is not behaving in any strange way. Many reports, however, have in the past identified accounts that _were_ clearly out to ruin the game for others. I am familiar with examples of accounts that completely dominated a match, only to resign or Ctrl-K their base at the near end of the game so they they would not receive any rating. Less extreme examples are more common, naturally, but occur frequently. 

Point 2. (Are people annoyed because of smurfing) and point 4 (why are they annoyed) seem asinine to me; we both know that smurfing is an issue and that people have issues with it. We do not need a formal inquiry to establish that this really is the case.

Re: 4)

0) No, we're not going to monetize alts. Such a proposal would encourage exactly the type of behavior that we're trying to prevent.
1) The system does not support this, replays do not contain the information that is required to make such decisions, and I'm not even going to start about the issues that come with false-positives.
2) This suggestion doesn't address any of the problems associated with smurfing. Smurfs will play their placement matches (smurfing all the while), and then enter the normal pool with their smurf rating. Also, I think you're significantly underestimating the amount of new players we get in any given time frame, and you need a decent pool to get a system like this working.
3) We currently give them a ban instead, and that seems to me to be significantly more effective. We're not going to subsidize moderating to the player base.
4) The rating system does not support giving back rating or making a game unrated after the fact. This also does not address smurfing itself, only the consequences of smurfing: it's treating symptoms, and we're interested in treating the cause.
5) I'd be in favour of adding 2FA to TMM, but I am not convinced that the problem of smurfing is large enough to require this at the moment. The rule against double accounts, and the frequent smurf-purges by the moderator team seem to work just fine. 

---

In summary: Smurfing is bad. The current method to address smurfing is sufficient and effective. We have this rule for good reasons, and no convincing arguments to removing it have been offered.

**Comment by Brutus5000** [comment]
### What are the downsides of multiple accounts

- We can no longer enforce the ownership check as one validated game could be used to verify everyone.
- Account explosions due to account issues. All of these appear more often than you could imagine:
  - I forgot my account name. I will make a new one.
  - I forgot my password. I will make a new account.
  - I have a cousin which is a friend of a friend who told he could resolve my issue X with a new account
- Rating manipulation:
  - It's a protection for lower rated players from getting crushed by higher rated players, that either constantly think they are overrated because they don't win 75% of the games or because they just want to get some easy wins)
  - Get into tournaments that have a rating boundary lower than the users current one.
- Circumvent the consequences of users prior actions.
  - Magically clean up the moderation history 
  - Avoid the foe list of other players
  - Go trolling on the forum
- Block/activity takeover/impersonate usernames from people who rotate their usernames
- In general blocking usernames, since we enforce usernames to be unique and it's deeply baked into FAF.

### What do other players gain if we allow one player to have a 2nd account?
- Nothing.

____

So the question remains what are the reasons that people want multiple accounts and are they valid?
- Popular Streamers are allowed to have a 2nd account, but they need to get "registered" (added to exempt rules in the database).
- I see no other valid reasons...?

**Comment by magge-faf** [comment]
Thank you for the notification and for initiating the discussion, fcaps.

I do not know if you remember the 'dark age' of FAF when anyone could create an account without any game-check on Steam/GOG — additionally, FAF had no tools to identify smurfs. Only since FAF has both of these issues been addressed, the toxicity has massively decreased over time. I know the FAF project since 2013, and it was a rough time back then, to put it _very_ mildly.

To be able to create multiple FAF accounts would only amplify the pattern of increased malicious activities. The very few benefits of smurfing don't justify all the trouble that is awaiting behind that. Popular streamers can ask for a second account, because otherwise they would get besieged instantly when they come online to prepare new content, or when they just want to play alone.

I would like to have 2FA as another layer of protection for all accounts necessary, but that is easier said than implemented.

To answer your initial question; Banning smurfing greatly enhances the community's positivity and security, holding extremely malicious players accountable for their actions.

**Comment by fcaps** [comment]
Thx for the input! and i take the L in not communicating my standpoint correctly and hurting others.

@magge-faf @IndexLibrorumProhibitorum @Brutus5000 
Just to be clear: I am **not** pro alt-account, i am for a better solution to counter smurfs and shitty (unbalanced) games (alt-account or not) without making it worse for others that currently rely on it.

@magge-faf 
there is 2FA study from blizzard about the topic, they finally decided against it, but they also made money out of alt accounts, so maybe not the best source. would be sad if we loosing a bunch of good players in this trade.  

> To answer your initial question; Banning smurfing greatly enhances the community's positivity and security, holding extremely malicious players accountable for their actions. 


i mean yeah, i got it.. don't let smurfs ruin other players fun, but there are other options besides banning alt accounts.
Alt-Account != Smurf, it's also used by players for some missing features in FAF.

----

There are some valid reasons against alt-accounts, but also some pro alt-accounts. Who am i to judge what will benefit the community without understanding the issue and implications completely, i saw an opportunity to make things better.

Don't know what you (the moderators) say about the current situation of smurf / alt-account detection and handling, but can't really imaging that sparks joy.

Here are more examples how i "could imagine" using an alt account:

1. My current rating in tmm 3v3 is not the same as my thermo/astro rating, when i play thermo, my rating is not reflecting my potential, usually resulting in crushing the enemy. when i play for example "valhalla", this is even a magnitude worst bc. they often playing better and i play worst than the global rating would suggest. A second account has helped there to create a true skill rating bound to the map-type/style without convincing others to balance it better without looking at the game quality.

2. Playing incognito is also something when trying to find new tactics, how often did i spy on other ppls training games to gather information about a new tactic they likely trying to apply to me.  Playing offline helped, but let's be real here, most are not aware they can even start faf offline and joining lobbies without the lobby-server discovery, or even how to disable replays. (some intentionally breaking replays with bugs as a workaround, but for every game and not only the trainings)

3. Starting the game without "being visible" to others, is not only a streamer thing, how often was i in the mood to "just play a game" without incoming messages, or being noticed. Yeah part time introverts exist.

Some, maybe even all could be solved via some features without making a "true" new account, maybe lets call the feature "light accounts", "accounts that don't have a login", "Incognito Profile" or "Guest Profile". Where some of your concerns and some of my could life together. <- THIS idea is not against smurfing, but pro "good" alt-account players.  


So for this:

>In summary: Smurfing is bad. The current method to address smurfing is sufficient and effective. We have this rule for good reasons, and no convincing arguments to removing it have been offered.

and this:

>What do other players gain if we allow one player to have a 2nd account?
Nothing.

When did you guys played the game more than just a couple of times in the last years :P 

I hope you understand the players that are using alt-accounts for non smurfing activities a bit better now, they even created gog accounts beside steam, bought the game twice, changed the uuid generator (or even used a dedicated VM), used a fallback internet if they had a static ip (some even using mobile hotspots), didn't interacted with friends on the platform and even used bugs just as a workaround for some missing features.

For real, i can even imagine some of them using their kids account (with a full history) after they stopped playing, or just saying "ahh on this game i helped my son" if they snuk into their account to play a casual game and got exposed.

Need to get potatoes for the long post.

**Comment by IndexLibrorumProhibitorum** [comment]
>Just to be clear: I am not pro alt-account

Then what is this conversation about? Because your two posts have explicitly argued _for_ allowing alts, and have not at all offered other methods of combating smurfing. 

Our position is that smurfing is bad, and disallowing people to make a second account in combination with active moderation to enforce that rule is effective in preventing smurfs. Your position seems to be that you believe people should be allowed to have a second account, regardless of the effect this has on the number of smurfs.

For the sake of argument, let's say I agree with you that people should have a second account. How do you propose smurfing is countered?

---

Also, I am missing what point you are trying to make with

>When did you guys played the game more than just a couple of times in the last years :P

**Comment by fcaps** [comment]
> Then what is this conversation about? Because your two posts have explicitly argued for allowing alts, and have not at all offered other methods of combating smurfing.

You are trying to combat smurfing by the one account rule, but this will not stop players form smurfing nor creating alt accounts. 

I have offered another solution, but maybe you missed it, here is a more complete list:
1. Detect smurfs by common patterns (game-data, account history, general behavior, etc.)
2. Make it less enjoyable to create a smurf account without making it hard for new players (x bot/unranked games before rated games etc.)
3. Let smurfs play against other smurfs or any other solution of "punishment" if we detected the account to be a potential smurf
4. Create a feedback rating-multiplier (if your teammates liked you, you getting more points, less if they hated you, or normal if they didn't voted pro/against you)
5.  etc.


> Also, I am missing what point you are trying to make with

> >When did you guys played the game more than just a couple of times in the last years :P

You are getting out of touch on how players are playing the game.

**Comment by IndexLibrorumProhibitorum** [comment]
>You are getting out of touch on how players are playing the game.

I'm active daily on the forum, on the discord, and play 2-3 games every day. Not quite sure what else you require to consider me an active player and in touch with the playerbase.

>You are trying to combat smurfing by the one account rule, but this will not stop players form smurfing nor creating alt accounts.

You have not made any convincing arguments that this is the case. In contrast, we've explained that we (the mods) believe the rule is working well. Magge has explained why we believe so. So far, your previous posts have explained why you believe the rule should not exist (focusing on the side-effects of the rule), but you have not explained why you believe the rule does not work to counter smurfs.

> I have offered another solution [to combat smurfing]
>1. Detect smurfs by common patterns (game-data, account history, general behavior, etc.)

This is already being done.

>2. Make it less enjoyable to create a smurf account without making it hard for new players (x bot/unranked games before rated games etc.)

This is already being done: smurf accounts get perm-banned, their main accounts get temp-banned. New players do not get banned.

>3. Let smurfs play against other smurfs or any other solution of "punishment" if we detected the account to be a potential smurf

The punishment we have settled on is a ban, see point 2.

>4. Create a feedback rating-multiplier (if your teammates liked you, you getting more points, less if they hated you, or normal if they didn't voted pro/against you)

This is nice to counter toxicity, but does not counter smurfing specifically. After all, smurf accounts are not made known to the moderator team with such a system, and players would still have to report smurfs. Which is the system that is now in place.

>5. etc

You can't just handwave in more solutions, lol.

---

Basically, what I need from you to continue this conversation is an explanation why you believe the current method to counter smurfs (prohibiting a second account, banning second accounts as they pop up) is not effective in countering smurfs.

**Comment by fcaps** [comment]
i'll stop it now, no worries. have a good one.

**Comment by Brutus5000** [comment]
> I have offered another solution, but maybe you missed it, here is a more complete list:
> 
> 1. Detect smurfs by common patterns (game-data, account history, general behavior, etc.)
> 2. Make it less enjoyable to create a smurf account without making it hard for new players (x bot/unranked games before rated games etc.)
> 3. Let smurfs play against other smurfs or any other solution of "punishment" if we detected the account to be a potential smurf
> 4. Create a feedback rating-multiplier (if your teammates liked you, you getting more points, less if they hated you, or normal if they didn't voted pro/
> You are getting out of touch on how players are playing the 

These are all useful suggestions that could be added as standalone features. Unfortunately we're not lacking ideas but people implementing them. 
There is so much more we could offer for different groups (there are dedicated user groups like coop players, tmm player, casual players, new joiners, streamers and they would all benefit from different things...)
